Lib/subprocess.py

```diff
@@ -446,6 +446,9 @@ def __repr__(self):
             args.append('stderr={!r}'.format(self.stderr))
         return "{}({})".format(type(self).__name__, ', '.join(args))
 
+    def __class_getitem__(cls, type):
+        return cls
```

Comment:
Here and below I would add a short docstring explaining why this is needed, and why it is implemented in such trivial way.

Reply:
What kind of docstring we should put? `"""For using this class as a generic, it implements PEP 560 __class_getitem__ method"""`?

Reply:
I think this should be something a bit more detailed. For example:
```python
def __class_getitem__(cls, type):
    """Provide minimal support for using this class as generic (for example in type annotations).

    See PEP 484 and PEP 560 for more details. For example, `CompletedProcess[bytes]` is a valid
    expression at runtime (type argument `bytes` indicates the type used for stdout). Note, no
    type checking happens at runtime, but a static type checker can be used.
    """
    return cls
```

Reply:
I see thanks.